src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -162,12 +162,22 @@ case class ArtifactOrProject(asString: String) {
       case _ => ArtifactOrProject(s"$asString-$str")
     }
   }
+
+  def addClassifier(c: Classifier): ArtifactOrProject = {
+    val str = c.asString
+    str match {
+      case "" => this
+      case _ => ArtifactOrProject(s"$asString-$str")
+    }
+  }
 }
 object ArtifactOrProject {
   implicit val ordering: Ordering[ArtifactOrProject] = Ordering.by(_.asString)
 }
 
 case class Subproject(asString: String)
+case class Classifier(asString: String)
+case class Packaging(asString: String)
```

Comment:
I think this. I think this is missing from #130 do you agree @roblg ?

Reply:
Yep, I ignored it in favor of just hard-coding the 5 non-`jar` dependencies we have because I was getting bogged down in scala , but it should definitely be supported for completeness.